assignments/lambda-classes.js

```diff
@@ -1 +1,256 @@
-// CODE here for your Lambda Classes
+//=====Global Functions=====
+
+function randomizeNumber(max) {
+	let posOrNeg = Math.random() < 0.2 ? -1 : 1;
+	let randomNumber = Math.floor(Math.random() * Math.floor(max));
+	return posOrNeg * randomNumber;
+}
+
+function gradingInstructor() {
```

Comment:
I really like the fact that you randomised instructor as well. you could have used your randomizer global function to randomly select an instructor.